### DISCUSSION TITLE
Pivot Table Slicer?

### DISCUSSION THREAD

**Discussion by stanps** [dfinke/ImportExcel#1582] [Q&A] [answered]
Greetings! :)

I was wondering if there's an option to add a Slicer to a pivot table?

If so, I would greatly appreciate someone pointing me in right the direction?  I've searched and can only find completed requests for it...and I don't see anything obvious in the available command line options.

TIA,
Stan

**Comment by dfinke** [comment, marked as answer]
Thanks for the question and using the module @stanps. Unfortunately, there is no support for slicers.

**Comment by LoanessyNew** [comment]
Greetings :)
Is there any way to create custom color for different slices in Pie chart via ImportExcel powershell ?

**Comment by dfinke** [comment]
No ready made function for that. When you use `Export-Excel` add `-PassThru`.

Now you have access to the XL programmatic object model. You can poke around there and find the "Drawing" collection, I believe.

This should let you navigate to a chart and get a reference. You need to inspect the methods on that to access the series and/or chart elements and then see how you can customize.
